websites/W/Wiki.gg/dist/metadata.json

```
@@ -0,0 +1,41 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Dark_Ville",
+		"id": "638080361179512853"
+	},
+	"service": "Wiki.gg",
+	"description": {
+		"en": "Platform operated by creators of gaming experiences. Flexible platform that adjusts to the needs of creators.",
+		"nl": "Platform beheerd door makers van spelervaringen. Flexibel platform dat zich aanpast aan de behoeften van de makers."
```

Suggestion from @QkeleQ10 
```suggestion
		"nl": "Wij brengen een gemeenschap van ontwikkelaars en editors samen om de best mogelijke ervaring voor gamers te creëren."
```